zubhub_frontend/zubhub/src/views/project_details/ProjectDetails.jsx

```diff
@@ -161,6 +176,13 @@ function ProjectDetails(props) {
     }
   }, [state.project.video]);
 
+  React.useEffect(() => {
+    if(enlarged_image_url.length > 0) {
+      let enlarged_image_index = project.images.map(image=>image.image_url).indexOf(enlarged_image_url)
+      handleSetState({ enlarged_image_index })
```

Comment:
I think its good to handle this in the place `enlarged_image_url` is being set, otherwise we would be causing multiple rerenders